src/sgame/sg_admin.cpp

```diff
@@ -1301,15 +1301,13 @@ static int admin_out( void *admin, char *str )
 
 	l = G_admin_level( a->level );
 
-	int lncol = Color::StrlenNocolor( l->name );
```

Comment:
What we want is to have `admin_level_maxname` non-color code points in the field. `Color::StrlenNocolor( l->name )` is the number of non-color code points in the name. So we want to add `admin_level_maxname - Color::StrlenNocolor( l->name )` spaces. So the width specifier should be bigger than the name's byte length by that amount. Therefore the fix is `lncol = strlen( l->name ) + ( admin_level_maxname - Color::StrlenNocolor( l->name ) )` (and revert the change to admin_level_maxname).